compiler/arm/README.md

```diff
@@ -0,0 +1,40 @@
+<!--
+Copyright (c) 2018, 2018 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OMR
+
+| Build | Status |
+| ---------------------- | -------------------- |
+| **Travis CI** | [![Travis Status](https://api.travis-ci.org/eclipse/omr.svg?branch=master)](https://travis-ci.org/eclipse/omr) |
+| **Travis CI Linux ARM Matrix Build** | [![Travis Status](https://travis-matrix-badges.herokuapp.com/repos/eclipse/omr/branches/master/4)](https://travis-ci.org/eclipse/omr) |
+ARM (32-Bit)
```

Comment:
An extra blank line is needed here. Otherwise, GitHub will interpret this as part of the table.

Reply:
Ok. Will fix up.